alua.c

```diff
@@ -542,21 +543,19 @@ bool lock_is_required(struct tcmu_device *dev)
 	return !!rhandler->lock;
 }
 
-static void *alua_lock_thread_fn(void *arg)
+static void alua_lock_work_fn(void *arg)
 {
 	struct tcmu_device *dev = arg;
 
 	tcmu_set_thread_name("alua-lock", dev);
 
 	/* TODO: set UA based on bgly's patches */
 	tcmu_acquire_dev_lock(dev, -1);
```

Comment:
Maybe we could check the rdev's `TCMUR_DEV_FLAG_STOPPING` state just after the `aio_wait_for_empty_queue()` which may take a long time, and then just skip acquiring the lock and returned.